CMakeLists.txt

```diff
@@ -741,9 +741,172 @@ add_dependencies(charmxi create_symlinks)
 # Converse
 add_subdirectory(src/QuickThreads)
 add_subdirectory(src/util/boost-context)
+
```

Comment:
I'd consider putting these into a separate file, maybe `cmake/converse.cmake`

Reply:
Would that require that all paths have `../` at the beginning?

Reply:
Probably, yes

Reply:
Wait, if you `include()` the cmake file, paths should still be relative to the main CmakeLists.txt, i.e., without `../`.